### PR TITLE
doc: Fix markdown formatting

### PR DESCRIPTION
**Pull request by gunar**
Lines were being joined making it hard to read.

### CODE REVIEW

File: doc/init.md [hunk 1]
```diff
@@ -53,11 +53,12 @@ Paths
 
 All three configurations assume several paths that might need to be adjusted.
 
-Binary:              `/usr/bin/bitcoind`
-Configuration file:  `/etc/bitcoin/bitcoin.conf`
-Data directory:      `/var/lib/bitcoind`
-PID file:            `/var/run/bitcoind/bitcoind.pid` (OpenRC and Upstart) or `/run/bitcoind/bitcoind.pid` (systemd)
-Lock file:           `/var/lock/subsys/bitcoind` (CentOS)
+    Binary:              /usr/bin/bitcoind
```
Comment (RandyMcMillan) [inline doc/init.md:56]: Binary:              /usr/bin/bitcoind
	Configuration file:  /etc/bitcoin/bitcoin.conf
	Data directory:      /var/lib/bitcoind
	PID file:            /var/run/bitcoind/bitcoind.pid (OpenRC and Upstart) or
                         /run/bitcoind/bitcoind.pip (systemd)
	Lock file:           /var/lock/subsys/bitcoind (CentOS)

Comment (RandyMcMillan) [reply doc/init.md:56]: I recommend 4 spaces for indenting.

**Review by RandyMcMillan — CHANGES_REQUESTED** [review]

File: doc/init.md (hunk 1 above)
Comment (zoltan-konder) [reply doc/init.md:56]: Meh, he just want to merge this to promote himself. He literally only changed indenting. I think markdown files isn't important at all. Puff up as "fix". :laughing:

**Comment by RandyMcMillan** [comment]
Additional suggested improvements:

https://github.com/RandyMcMillan/bitcoin-core/tree/gunar-patch-1

File: doc/init.md (hunk 1 above)
Comment (RandyMcMillan) [reply doc/init.md:56]: I disagree - I wish more people reviewed documentation and submitted improvements.
This should be encouraged.

Comment (zoltan-konder) [reply doc/init.md:56]: You just disagree because you are an American terrorist at the @usnavy. The documentation is well written, there is nothing to fix. Adding a few spaces to get something merged in GitHub is absurd. You can run now to the Pentagon and cry about Zoltán Kondér.

What the hell a @usnavy has to do with Bitcoin anyway?

Comment (gunar) [reply doc/init.md:56]: This is formatting I'm looking to fix (below). It's indeed my first contribution albeit small. Looking forward to making more meaningful ones as I get more comfortable with the codebase.

image

Comment (RandyMcMillan) [reply doc/init.md:56]: I have linked to some suggested changes -> 

> Additional suggested improvements:
> 
> https://github.com/RandyMcMillan/bitcoin-core/blob/gunar-patch-1/doc/init.md

add those..

Comment (zoltan-konder) [reply doc/init.md:56]: @gunar :+1: I think there should be more changes than just adding spaces. At least write something..

**Review by promag — COMMENTED** [review]
Could also fix https://github.com/bitcoin/bitcoin/blob/58a1dc6b355c4964a6d6b99eaa1edcc40369e104/doc/init.md#macos

Edit: @RandyMcMillan already pointed this.

**Comment by gunar** [comment]
Suggestions merged and commits squashed. Thank you.

**Comment by laanwj** [comment]
ACK dcab5e813348e4a01517a99e129c0a27e6224298

For next time, when you make changes aimed at formatting on the website: it's useful if you link to the rendered documents so people can easily compare. E.g.

Before: https://github.com/bitcoin/bitcoin/blob/a2a03c3ca94b1cdd279ac09f2a81e04d262586fd/doc/init.md#paths
After: https://github.com/bitcoin/bitcoin/blob/dcab5e813348e4a01517a99e129c0a27e6224298/doc/init.md#paths

File: doc/init.md [hunk 2]
```diff
@@ -53,11 +53,12 @@ Paths
...
-Binary:              `/usr/bin/bitcoind`
-Configuration file:  `/etc/bitcoin/bitcoin.conf`
-Data directory:      `/var/lib/bitcoind`
-PID file:            `/var/run/bitcoind/bitcoind.pid` (OpenRC and Upstart) or `/run/bitcoind/bitcoind.pid` (systemd)
-Lock file:           `/var/lock/subsys/bitcoind` (CentOS)
+	Binary:              /usr/bin/bitcoind
+	Configuration file:  /etc/bitcoin/bitcoin.conf
+	Data directory:      /var/lib/bitcoind
+	PID file:            /var/run/bitcoind/bitcoind.pid (OpenRC and Upstart) or
+                         /run/bitcoind/bitcoind.pip (systemd)
```
Comment (MarcoFalke) [inline doc/init.md:60]: What is the point of mixing tabs and spaces? Why is the file extension changed to pip?

File: doc/init.md [hunk 3]
```diff
@@ -4,11 +4,11 @@ Sample init scripts and service configuration for bitcoind
...
-    contrib/init/bitcoind.service:    systemd service unit configuration
-    contrib/init/bitcoind.openrc:     OpenRC compatible SysV style init script
-    contrib/init/bitcoind.openrcconf: OpenRC conf.d file
-    contrib/init/bitcoind.conf:       Upstart service configuration file
-    contrib/init/bitcoind.init:       CentOS compatible SysV style init script
+	contrib/init/bitcoind.service:    systemd service unit configuration
+	contrib/init/bitcoind.openrc:     OpenRC compatible SysV style init script
+	contrib/init/bitcoind.openrcconf: OpenRC conf.d file
+	contrib/init/bitcoind.conf:       Upstart service configuration file
+	contrib/init/bitcoind.init:       CentOS compatible SysV style init script
```
Comment (MarcoFalke) [inline doc/init.md:11]: Why is this changed?

**Review by maflcko — CHANGES_REQUESTED** [review]
this changes the filename from pid to pip. why?

File: doc/init.md (hunk 3 above)
Comment (gunar) [reply doc/init.md:11]: image

Comment (MarcoFalke) [reply doc/init.md:11]: That is a screenshot of the rendered changes of the second hunk. The first hunk (`contrib/init`) renders fine already.

File: doc/init.md (hunk 2 above)
Comment (gunar) [reply doc/init.md]: The file extension change was a typo. Fixed now.

Mixing tabs and spaces is what https://github.com/bitcoin/bitcoin/blob/master/doc/tor.md#L24-L27 does. Do you think we should change it everywhere?

File: doc/init.md (hunk 3 above)
Comment (gunar) [reply doc/init.md:11]: My bad—this is for tabs/spaces inconsistency consistency. Happy to go one way or the other.

Re: https://github.com/bitcoin/bitcoin/pull/21075#discussion_r570063070

Comment (laanwj) [reply doc/init.md:11]: Yes, please don't use tabs. If it wasn't clear yet, we use spaces for indentation only in this project.

Comment (gunar) [reply doc/init.md]: Cool, thanks for the info. I've just replaced tabs for spaces in all other `doc/*.md` files too: [`e1604b3` (#21075)](https://github.com/bitcoin/bitcoin/pull/21075/commits/e1604b3d50dca3291a432be59cfd03c0e846e7b2)

**Comment by RandyMcMillan** [comment]
ACK e1604b3d50dca3291a432be59cfd03c0e846e7b2

This commit renders cleanly on gh-flavored md and @MacDownApp 

build-unix.md [before](https://github.com/bitcoin/bitcoin/blob/ae749d12ddbaf592fbdb65d98ca35a0ff5566992/doc/build-osx.md)
build-unix.md [after](https://github.com/bitcoin/bitcoin/blob/e1604b3d50dca3291a432be59cfd03c0e846e7b2/doc/build-unix.md)

init.md [before](https://github.com/bitcoin/bitcoin/blob/ae749d12ddbaf592fbdb65d98ca35a0ff5566992/doc/init.md)
init.md [after](https://github.com/bitcoin/bitcoin/blob/e1604b3d50dca3291a432be59cfd03c0e846e7b2/doc/init.md)

tor.md [before](https://github.com/bitcoin/bitcoin/blob/ae749d12ddbaf592fbdb65d98ca35a0ff5566992/doc/tor.md)
tor.md [after](https://github.com/bitcoin/bitcoin/blob/e1604b3d50dca3291a432be59cfd03c0e846e7b2/doc/tor.md)

File: doc/tor.md [hunk 4]
```diff
@@ -21,39 +21,39 @@ information in the debug log about your Tor configuration.
...
+                    other P2P nodes.
+
+    -onlynet=onion  Make outgoing connections only to .onion addresses. Incoming
+                    connections are not affected by this option. This option can be
+                    specified multiple times to allow multiple network types, e.g.
+                    ipv4, ipv6 or onion. If you use this option with values other
+                    than onion you *cannot* disable onion connections; outgoing onion
+                    connections will be enabled when you use -proxy or -onion. Use
+                    -noonion or -onion=0 if you want to be sure there are no outbound
+                    onion connections over the default proxy or your defined -proxy.
```
Comment (jonatack) [inline doc/tor.md:52]: Why were all of these changed, rather than only the lines with tabs?

FWIW I proposed to do just that in this file a few months ago and was asked not to because it was unnecessary churn.

Comment (jonatack) [reply doc/tor.md:52]: :man_shrugging:
